### PR TITLE
Spanish update

### PR DESCRIPTION
**Pull request by KutonnoZer0**
Translate for:
- **/main.txt**  
          "Unassigned Devices"

- **/settings.txt** 
          "Add Route"

- **Community Apps/apps.txt**
          "You have been logged out"
          "A note about translations"
          "Click here to view the language changelog"

-**Auto-Update Apps**
-**Config Edit**

### CODE REVIEW

File: CA Config File Edit/configedit.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-Editing=
-Generally, it is not recommended to change the format=
-Credits=
-No Backup File Found=
-File Format=
+Select File=Seleccciona un archivo
+Error reading file=Error al leer el archivo
+No Backup File Found=No se encontró archivo de respaldo
+Select a file to edit=Seleccione un archivo para editar
+note editing a binary file could have unintended consequences=nota editar un archivo binario podría tener consecuencias no deseadas
```
[the remark this answers is not in the document]
Comment (KutonnoZer0) [reply CA Config File Edit/configedit.txt:5]: What do you think about? 
      "Nota: Editar un archivo binario podría tener consecuencias no deseadas."

File: CA Config File Edit/configedit.txt [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+No file selected=Ningún archivo seleccionado
+Backup File=Archivo de respaldo
+Editing=Editando
+Generally, it is not recommended to change the format=En general, no se recomienda cambiar el formato
```
[the remark this answers is not in the document]
Comment (KutonnoZer0) [reply CA Config File Edit/configedit.txt:9]: Sound good but when you use "Generalmente" is like "Usually".
"En General" is like you are strongly suggesting something. is hard to explains by typing.

File: CA Config File Edit/configedit.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (sonoseco) [reply CA Config File Edit/configedit.txt:5]: Although we would be adding a special character that’s not in the original language, I think that using `Nota:` makes more sense.

[the remark this answers is not in the document]
Comment (spencerjunraid) [reply CA Config File Edit/configedit.txt:5]: Incluí el colon después de "nota"

**Pull request merged**